dll/cpl/appwiz/addons.c

```diff
@@ -321,8 +321,23 @@ static HRESULT WINAPI InstallCallback_OnStopBinding(IBindStatusCallback *iface,
 static HRESULT WINAPI InstallCallback_GetBindInfo(IBindStatusCallback *iface,
         DWORD* grfBINDF, BINDINFO* pbindinfo)
 {
-    /* FIXME */
+    DWORD cbSize;
+
+    cbSize = pbindinfo->cbSize;
+    ZeroMemory(pbindinfo, cbSize);
+    pbindinfo->cbSize = cbSize;
+
     *grfBINDF = 0;
```

Comment:
Note:
Wine is now using `BINDF_ASYNCHRONOUS`. Would we want to port that change (too), here or separately?

Reply:
`BINDF_ASYNCHRONOUS` needs some mechanism. I don't implement it yet.